core/tests/performance_framework/perf_services.py

```diff
@@ -206,12 +206,31 @@ def get_page_timings_from_cached_session(self, page_url):
             page_session_stats=None, page_session_timings=page_session_timings)
 
     def _wait_until_page_load_is_finished(self, time_duration_secs=None):
+        """Waits for the complete page to load, otherwise XHR requests
+        made post initial page load will not be recorded.
+
+        Args:
+            time_duration_secs: int. Waiting time for page load. Defaults to
+                None.
+        """
         # Waits for the complete page to load, otherwise XHR requests
         # made post initial page load will not be recorded.
         time.sleep(time_duration_secs or self.DEFAULT_WAIT_DURATION_SECS)
 
     def _setup_proxy_server(self, downstream_kbps=None, upstream_kbps=None,
                             latency=None):
+        """Sets up a browsermobproxy server.
+
+        Args:
+            downstream_kbps: int. The downstream speed in kbps. Defaults to
+                None.
+            upstream_kbps: int. The upstream speed in kbps. Defaults to None.
+            latency: int. The latency of the server. Defaults to None.
+
+        Returns:
+            tuple(server, proxy): A tuple consisting of the server and the
```

Comment:
I don't think "server" and "proxy" are the actual type names; ditto for "driver" below and throughout the remainder of this file. Could you please make sure the types are correct? For instances of classes it should be the class name (so, in this case, it should be Server for the first arg, since it's an instance of browsermobproxy.Server).

Don't just change everything to start with uppercase, though -- make sure to actually figure out what class it is an instance of. When resolving this comment, please also include (in your comment) a reference to where the driver class is defined.

Thanks!